### ISSUE TITLE
Single node: Improve messaging for undefined docker-workers section

### ISSUE THREAD

**Issue by rikukissa** [opencrvs/opencrvs-core#6918]
Currently, ansible playbooks fail with a no-so-clear error message if the user tries to provision an environment without a [`docker-workers`](https://github.com/opencrvs/opencrvs-countryconfig/blob/develop/infrastructure/server-setup/production.yml#L44-L49) block in their inventory file. 

![Image](https://github.com/opencrvs/opencrvs-core/assets/1206987/ee68506d-42d8-421f-8e02-e744d3683c7a)

### Task
- [x] **Documentation**: [On this documentation page](https://documentation.opencrvs.org/setup/3.-installation/3.3-set-up-a-server-hosted-environment#server-environments) add a new clear warning / suggestion / recommendation for countries to always deploy production environment to a >2 server clusters. This is so that there is no downtime or data loss in the case of one of the servers going down.
- [x] **Improved messaging:** introduce a new configuration variable in the Ansible inventory file for machines labeled as `docker-manager-first`. The variable's purpose is to force the user to explicitly state they are deploying the environment as a single-node setup. The variable can for instance be called `single-node: true`. In our country config example inventories, this should be true for QA and staging but false for production. There should be a comment in the production file over this variable, stating that we do not recommend single node setups for production use. The ansible playbook fails with an error if this variable is not set and there are no docker-workers. The error message needs to be clear enough for the user to know exactly how to fix the issue.

**Comment by Zangetsu101** [comment]
@rikukissa @adskyiproger The ticket mentions failing the workflow if production is being configured as a single node. Now let's say a country still want to do it, in that case they would need to make changes to the playbook to get past the error. Is that how we want it to be or should it be a warning instead of an error?

**Comment by adskyiproger** [comment]
On my opinion it should be warning, not error

**Comment by Zangetsu101** [comment]
Yeah agreed, we don't want people to unnecessarily make changes to the playbook.

**Comment by Zangetsu101** [comment]
Wait I might have gotten a bit confused earlier, @alsmk correct me if I am wrong, we error only if `single_node = false` & `docker-worker` is not defined right? If `single_node = true` then we don't

If that is the case then I think we are fine

**Comment by alsmk** [comment]
> Wait I might have gotten a bit confused earlier, [@alsmk](https://github.com/alsmk) correct me if I am wrong, we error only if `single_node = false` & `docker-worker` is not defined right? If `single_node = true` then we don't.
> 
> If that is the case then I think we are fine

Yes, we are not raising error if `single_node: true`
I think we are fine to proceed
